src/integer/poly_over_z/set.rs

```diff
@@ -78,16 +76,16 @@ mod test_set_coeff {
         assert!(poly.set_coeff(i8::MIN, 2).is_err());
     }
 
-    /// Ensure that coefficients up to 2^15 -1 work
+    /// Ensure that big coefficients work.
     #[test]
-    fn set_max_coeff() {
+    fn set_coeff_big() {
```

Comment:
I would prefer large over big, it is used more in other tests and is more formal. I also added this to the refactoring.